### ISSUE TITLE
Feature request: add option to restore default minimap viewport color

### ISSUE THREAD

**Issue by simonhaenisch** [equinusocio/material-theme#1018]
[This commit](https://github.com/equinusocio/material-theme/commit/0aa5b69) makes the minimap viewport use the accent color for the minimap viewport. It really distracts me, I liked the unobtrusive gray a lot better. I use the tomato accent and it just jumps into the eye all the time.

Can you please add an option like `material_theme_no_accent_for_minimap`?

**Comment by equinusocio** [comment]
You can just edit this by yourself inside your local installation. Anyway i've reduced the minimap viewport opacity.

**Comment by simonhaenisch** [comment]
won't this be overwritten on each upgrade?

**Comment by equinusocio** [comment]
Yes, I've also fixed `"always_show_minimap_viewport": true`. To change the viewport color just make a `Material Theme` folder inside your st Packages directory. Put inside a new file called `Material-Theme-**Yourvariant.sublime-theme`(ex if you're using Darker scheme the file should be named `Material-Theme-Darker.sublime-theme`)  and add this code:

```json
    {
      "class": "minimap_control",
      "viewport_color": [RR, GG, BB, 80],
      "viewport_opacity": 0.5,
    },
```

Replace RR,GG,BB value with an RGB color, you can also change the viewport opacity. Then save and restart sublime text.

**Comment by simonhaenisch** [comment]
It doesn't work for me (on Windows). It seems to overwrite the complete Package (all UI elements turn black) instead of just overwriting the `minimap_control` setting. I'll wait for your fix then.

**Comment by equinusocio** [comment]
Mmm try to rename the `Material Theme` folder to `Z Material Theme`

**Comment by simonhaenisch** [comment]
Ok, got it, I just placed it like `Packages/User/Material-Theme-Darker.sublime-theme`, guess that makes the most sense, as the `User` folder is for overwriting default settings with user settings. Anyway it works as long as it is in either `Packages` or a sub-directory inside `Packages`. Naming a folder like a package (from `Installed Packages`) seems to overwrite the package with the one in the folder.

Also I had to put it like

```json
[
    {
      "class": "minimap_control",
      "viewport_color": [255, 255, 255, 40],
      "viewport_opacity": 0.5,
    }
]
```

inside an array, otherwise the console told me
> `top level value must be an array`

Thank you very much for your help!
